From a859377ee3b3c700ca6c5cd878e7c2f1a60143a6 Mon Sep 17 00:00:00 2001
From: Ioana Radulescu <ruxandra.radulescu@nxp.com>
Date: Tue, 29 Aug 2017 17:49:16 +0300
Subject: [PATCH 509/706] staging: fsl-dpaa2/eth: Add DCB Kconfig

Add a Kconfig option for enabling Data Center Bridging (DCB)
support in the Ethernet driver. This is needed for priority
flow control support.

Signed-off-by: Ioana Radulescu <ruxandra.radulescu@nxp.com>
(cherry picked from commit bf0a45e623e18a6bdedefe27c6a81fb9163bf710)
Signed-off-by: Zumeng Chen <zumeng.chen@windriver.com>
---
 drivers/staging/fsl-dpaa2/Kconfig | 10 ++++++++++
 1 file changed, 10 insertions(+)

diff --git a/drivers/staging/fsl-dpaa2/Kconfig b/drivers/staging/fsl-dpaa2/Kconfig
index 3085b546e9a9..c514875fce29 100644
--- a/drivers/staging/fsl-dpaa2/Kconfig
+++ b/drivers/staging/fsl-dpaa2/Kconfig
@@ -51,6 +51,16 @@ config FSL_DPAA2_ETH_DEBUGFS
 	default n
 	---help---
 	  Enable advanced statistics through debugfs interface.
+
+config FSL_DPAA2_ETH_DCB
+        bool "Data Center Bridging (DCB) Support"
+        default n
+        depends on DCB
+        ---help---
+          Say Y here if you want to use Data Center Bridging (DCB) features
+          (PFC) in the driver.
+
+          If unsure, say N.
 endif
 
 source "drivers/staging/fsl-dpaa2/mac/Kconfig"
-- 
2.17.1

